---
std_msgs/Int16MultiArray output